tests/unit/data/lumen/lumen8_result.json

```diff
@@ -0,0 +1,40 @@
+[
+    {
+        "account": "Customer",
+        "circuits": [
+        {
+            "circuit_id": "123456789",
+            "impact": "OUTAGE"
+        },
+        {
+            "circuit_id": "123456789",
+            "impact": "OUTAGE"
+        }
+        ],
+        "end": 1679479200,
+        "maintenance_id": "12345678",
+        "stamp": 1678379003,
+        "start": 1679457600,
+        "status": "IN-PROCESS",
```

Comment:
Not something you've changed, but it seems odd to me that these are marked as `IN-PROCESS` rather than `CONFIRMED`. I'm not necessarily asking for any changes here, just making an observation. :-)